### PR TITLE
added docker 

### PR DESCRIPTION
**Pull request by msahraeian**
Added docker file, a docker test run, and information in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ doi: https://doi.org/10.1101/393801](https://doi.org/10.1101/393801)
 
 ## Table of Contents
 **[Availability](#availability)**<br>
+**[NeuSomatic Docker Image](#neusomatic-docker-image)**<br>
+**[Requird Inputs](#requird-inputs)**<br>
```
Comment (marghoob) [inline README.md]: Requird -> Required

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ENV PYTORCH_VERSION 0.3.1
+ENV TORCHVISION_VERSION 0.2.0
+ENV CMAKE_VERSION 3.12.1
+ENV PYBEDTOOLS_VERSION 0.7.10
+ENV PYSAM_VERSION 0.14.1
+ENV SAMTOOLS_VERSION 1.7
+ENV TABIX_VERSION 0.2.5
+ENV BEDTOOLS_VERSION 2.27.1
+ENV BIOPYTHON_VERSION 1.68
+ENV GCC_VERSION 5
```
Comment (marghoob) [inline docker/Dockerfile:16]: 👍 for adding versions in the beginning for readability.

File: test/docker_test.sh [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+test_dir=$(pwd -P)
+
+mkdir -p example
+cd example
+if [ ! -f Homo_sapiens.GRCh37.75.dna.chromosome.22.fa ]
+then
+	if [ ! -f Homo_sapiens.GRCh37.75.dna.chromosome.22.fa.gz ]
+	then
+		docker run -v ${test_dir}:/mnt -u $UID --memory 30G  msahraeian/neusomatic:0.1.1 /bin/bash -c \
```
Comment (marghoob) [inline test/docker_test.sh:12]: Have you already pushed to the Docker hub?

File: README.md [hunk 4]
```diff
@@ -57,6 +59,16 @@ conda install pysam=0.14.1 pybedtools=0.7.10 samtools=1.7 tabix=0.2.5 bedtools=2
 ```
 g++ 5.4.0 can also be obained as `sudo apt-get install gcc-5 g++-5`.
 
+## NeuSomatic Docker Image
+
+The docker image with all the packages installed (CPU-only) can be found at https://hub.docker.com/r/msahraeian/neusomatic/ 
+
+To use GPU (in `train.py` and `call.py` steps), you should use conda environment to locally install requried packages as shown above.
```
Comment (marghoob) [inline README.md]: requried --> required

File: test/docker_test.sh (hunk 3 above)
Comment (msahraeian) [reply test/docker_test.sh:12]: yes

**Comment by marghoob** [comment]
This is good to be merged. 👍

**Comment by marghoob** [comment]
Will you create a new release for this?

**Comment by msahraeian** [comment]
Not now. Later on after we have more pull requests merged. maybe next week

**Pull request merged**